### PR TITLE
0.115.1

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by dsherret — APPROVED** [review]
Not a patch release?

**Comment by bartlomieju** [comment]
> Not a patch release?

Sure I can do patch

File: version.ts [hunk 1]
```diff
@@ -5,4 +5,4 @@
  * the cli's API is stable. In the future when std becomes stable, likely we
  * will match versions with cli as we have in the past.
  */
-export const VERSION = "0.115.0";
+export const VERSION = "0.116.0";
```
Comment (dsherret) [inline version.ts:8]: Missing change.

Comment (bartlomieju) [reply version.ts]: I hate safe-write in editors... -.-

**Pull request merged**